app/src/main/AndroidManifest.xml

```diff
@@ -142,7 +142,7 @@
                 android:resource="@xml/entity_widget_info" />
         </receiver>
 
-        <receiver android:name=".widgets.media_player_controls.MediaPlayerControlsWidget" android:label="@string/widget_media_player_description"
+        <receiver android:name=".widgets.mediaplayer.MediaPlayerControlsWidget" android:label="@string/widget_media_player_description"
```

Comment:
This will be a breaking change for media player widgets, we experienced this after https://github.com/home-assistant/android/pull/1000 was merged a while back

Reply:
Just need to make sure it makes it into the changelog.

Reply:
agreed, this note is a reminder to myself 😂